ika/driver.py

```diff
@@ -282,23 +282,123 @@ async def reset(self):
 class ShakerProtocol:
     """Protocol for communicating with an orbital shaker.
 
+    RS-232 Information
+        - Transmission procedure: asynchronous character transmission in start-stop mode
+        - Type of transmission: full duplex
+        - 1 start bit; 7 character bits; 1 parity bit (even); 1 stop bit
+        - Transmission speed: 9600 bit/s
+
     Command syntax and format from the manual:
         - commands and parameters are transmitted as capital letters
         - commands and parameters including successive parameters are separated by at least
           one space (hex 0x20)
         - each individual command (including parameters and data and each response are terminated
-          with Blank CR LF (hex 0x20 hex 0x0d hex 0x0A) and have a maximum length of 80 characters
+          with Blank CR LF (hex 0x0d hex 0x0A) and have a maximum length of 80 characters
```

Comment:
Good catch; the manual is incorrect.  (different manuals say different things, which is annoying).

However, your update isn't fully correct, either.

Blank CR LF != 0x0d 0x0A

Reply:
So which is correct, left or right? Do I just need to trial and error to figure it out? (The Vacstar manual says "CRLF (code: hex 0x20 hex 0x0d hex 0x0A)" which is perhaps the most confusing of all)

Reply:
Yes, experiment and find out.